update the minimum version policy (#4907)
pydata/xarray: doc/installing.rst

  numpy >=1.17.
- **all other libraries:** 12 months

> The above should be interpreted as *the most recent minor version (X.Y) initially
> published more than N months ago*. Patch versions (x.y.Z) are not pinned, and only the

Reviewer: How about this? (from [here](https://github.com/numpy/numpy/pull/14086#issuecomment-649762395))

```suggestion
This means the latest minor (X.Y) version from N months prior.
Patch versions (x.y.Z) are not pinned, and only the
```

Reviewer 2: The suggested change is very clear to me.

Author: if anyone feels a particular wording is much easier to understand feel free to push to this branch
